src/utility-bar/bar.tsx

```diff
@@ -55,6 +69,18 @@ export const ShellUtilityBar = (): React.JSX.Element => {
 
 	const { reset } = useTracker();
 
+	const actions = useActions<undefined, AccountMenuAction>(undefined, ACTION_TYPES.ACCOUNT_MENU);
+	const accountMenuItems = useMemo(
+		(): DropdownItem[] =>
+			actions
+				.sort((a, b) => a.position - b.position)
+				.map((action) => ({
+					onClick: action.execute,
+					...action
```

Comment:
Extract the execute and the position fields, which don't belong to the dropdown, to prevent passing them to the component (and so to the dom)
```suggestion
				.map(({ .execute, position: _position, ...action }) => ({
					onClick: execute,
					...action
```